lib/version_utils.pm

```diff
@@ -576,7 +576,7 @@ sub get_os_release {
     my ($go_to_target, $os_release_file) = @_;
     $go_to_target    //= '';
     $os_release_file //= '/etc/os-release';
-    my %os_release = script_output("$go_to_target cat $os_release_file") =~ /^(\S+)="?([^"\r\n]+)"?$/gm;
+    my %os_release = script_output("$go_to_target cat $os_release_file") =~ /^([^#]\S+)="?([^"\r\n]+)"?$/gm;
```

Comment:
can you give more certain examples when something is commented out ? Looks suspicious to me. If something is commented the person who did this basically sending a message to everyone "please ignore this line" and usually he/she had a good reason to ask this. And we ignoring this request which might bring to unexpected results ...

Reply:
Here you go :) 

```
❯  cat /etc/os-release                             
NAME="openSUSE Tumbleweed"
# VERSION="20201011"
ID="opensuse-tumbleweed"
ID_LIKE="opensuse suse"
VERSION_ID="20201011"
PRETTY_NAME="openSUSE Tumbleweed"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:tumbleweed:20201011"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
DOCUMENTATION_URL="https://en.opensuse.org/Portal:Tumbleweed"
LOGO="distributor-logo"
```

Reply:
`[^#]` is reverse lookup. So lines which starts with hash would not be included. As it was before it would match strings with `#` or without anyway. This also was breaking the mapping in the next line. is it clear now?

Reply:
@asmorodskyi can i resolve this or do you have any additional comment?